Sources/Extensions/MapKit/MKMapSnapshotterExtensions.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  MKMapSnapshotterExtensions.swift
+//  SwifterSwift
+//
+//  Created by Amit Bhavsar on 06/02/19.
+//  Copyright © 2019 SwifterSwift
+//
+
+
+#if canImport(MapKit)
+import MapKit
+
+#if canImport(Kingfisher)
+import Kingfisher
+#endif
+
+#if !os(watchOS)
+// MARK: - Methods
+@available(tvOS 9.2, *)
+public extension CLLocationCoordinate2D {
+    
+    
+    /// SwifterSwift: Return the image of map region
+    ///
+    /// Example:
+    ///     let location = CLLocationCoordinate2D(latitude: 37.332078, longitude: -122.02962)
+    ///     location.snap(withSize: imgvMap.frame.size) { [weak self] image in
+    ///         if let _ = image {
+    ///             self?.imgvMap.image = image
+    ///         }
+    ///     }
+    ///
+    /// - Parameters:
+    ///   - size: size of the map image which will be displayed
+    ///   - completion: It will return the required map image
+    public func snap(withSize size:CGSize, completion:@escaping (UIImage?) -> Void) {
+        let mapSnapshotOptions = MKMapSnapshotter.Options()
```

Comment:
Pass this as parameter with default options instead of assuming the region with delta, `scale`, `showBuildins`, `showPointsOfInterest`.